### PR TITLE
README: Fixup instructions for running Orange

### PR DESCRIPTION
**Pull request by irgolic**
##### Issue
https://github.com/biolab/orange3-text/issues/623
I'm assuming the root of this issue is having installed Orange with pip.

##### Description of changes
Add a Windows warning next to the pip installation instructions.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ajdapretnar** [comment]
This doesn't fix biolab/orange3-text#623. The OP had a Linux installation, which was segfaulting (likely because of WebView). To solve this, one would have to look into Linux installation instruction and perfect them.

**Comment by markotoplak** [comment]
@irgolic, thanks, but in biolab/orange3-text#623 one person also had a problem on Linux, with both pip and conda. I suspect Qt...

@ajdapretnar, why wouldn't Orange be pip-installable on Windows? Did anyone try it recently? I understand sometimes you could have some libraries that are not compiled, but these are getting rarer.

I think we should keep directing users towards our packages, as for the developers, well, we can let them choose. If they have compilers properly set up I do not see pip being a problem...

**Comment by ajdapretnar** [comment]
> @ajdapretnar, why wouldn't Orange be pip-installable on Windows? Did anyone try it recently? I understand sometimes you could have some libraries that are not compiled, but these are getting rarer.

Probably no one has tested it in a while, but numpy+mkl used to be an issue. As for Text (the issue was reported on Text), it cannot be installed on Windows due to gensim (I think).

**Comment by markotoplak** [comment]
Now I remembered: both Orange and Orange3-Text can work on Windows normally if pip installed (provided compilers are there, maybe even if they are not, depends on the libraries) otherwise our tests would fail (our tests use pip for installation).

**Comment by irgolic** [comment]
So what exactly are the prerequisited for a pip installation to fail?

**Comment by markotoplak** [comment]
In theory )and controlled practice), `pip` into a fresh environment should work well and actually does. It is what we test.

In practice, some packages may only have source packages on PyPI, which would require people to have their compiler chain installed. Most Windows users are not programmers, so they don't. Hence they could see errors because compilers are not found. This can also be a problem on Macs and Linux, but installing a compiler is easier (also, users are different).

I do not know what the current state on packages requiring compilation is, exactly... There was a period when some crucial Windows packages lacked binary packages, but nowadays, most of them actually do have them.

**Comment by janezd** [comment]
Decision at discussion: remove instructions for pip, which make an impression this is easy and works; instead say only that you can install Orange with pip, but you'll need compilers etc, and ... have fun.

File: README.md [hunk 1]
```diff
@@ -56,49 +56,32 @@ conda install orange3-<addon name>
...
-and run either `orange-canvas` or `python3 -m Orange.canvas`. 
-
-Add `--help` for a list of program options.
-
-Starting up for the first time may take a while.
 
 ### Installing with pip
 
-To install Orange with pip, run the following.
+We recommend using our [standalone installer](https://orange.biolab.si/download) or conda, but Orange is also installable with pip. On Windows, you will also need Microsoft Visual Studio Build Tools.
```
Comment (thocevar) [inline README.md:62]: You will need a C/C++ compiler (on Windows we suggest using Microsoft Visual Studio Build Tools).

**Comment by janezd** [comment]
@JakaKokosar are this problems with CI related to the experiment with rebasing?

**Comment by JakaKokosar** [comment]
Yes, they are. I will try to finish this as soon as possible :)

**Pull request merged**